src/main/java/emu/grasscutter/game/quest/GameMainQuest.java

```diff
@@ -51,7 +52,7 @@ public class GameMainQuest {
     @Getter private ParentQuestState state;
     @Getter private boolean isFinished;
     @Getter List<QuestGroupSuite> questGroupSuites;
-
+    @Getter @Setter private int rewardIndex = 0;
```

Comment:
should this maybe get a proper setter that also calls save?
 Or do you think its not importent due to rewinds or something similar?